Improve documentation (#28)
ambanum/CGUs: README.md

Tracks and makes visible all changes to the Terms Of Service of online service providers.

> Suit et rend visibles les modifications des Conditions Générales d'Utilisations des principaux fournisseurs de services en ligne.

> ## Installation

Reviewer: ```suggestion
## Installing
```

> This API is built with [Node](https://nodejs.org/en/). You will need to [install it](https://nodejs.org/en/download/) to run this application.

Reviewer: ```suggestion
This module is built with [Node](https://nodejs.org/en/). You will need to [install Node](https://nodejs.org/en/download/) to run it.
```

Clone the repository and install dependencies:

```sh
git clone https://github.com/ambanum/CGUs.git
cd CGUs
npm install
```

## Usage

> To get lastest versions of service providers' terms:

Reviewer: ```suggestion
To get the latest versions of all service providers' terms:
```

```
npm start
```

> The lastest version will be available in `/data/sanitized/<service_provider_name>/<document_type>.md`.

Reviewer: ```suggestion
The latest version will be available in `/data/sanitized/$service_provider_name/$document_type.md`.
```

To hourly update documents:

```
npm run start:scheduler
```

> ## Add a service provider

Reviewer: ```suggestion
## Adding a service provider
```

> In the folder `providers`, create a JSON file with the name of the service provider who want to add with following structure:

Reviewer: ```suggestion
In the folder `providers`, create a JSON file with the name of the service provider you want to add, with the following structure:
```

```json
{
  "serviceProviderName": "<the public name of the service provider>",
  "documents": {
    "<document type>": {

>       "url": "<the url where the document can be found>",

Reviewer: ```suggestion
      "url": "<the URL where the document can be found>",
```

>       "contentSelector": "<the css selector on the meaningful content of the document>",

Reviewer: ```suggestion
      "contentSelector": "<a CSS selector that targets the meaningful part of the document, excluding elements such as headers, footers and navigation>",
```

    }
  }
}
```

> For document type [there is a list of available documents types](/src/documents_types.js)

Reviewer: ```suggestion
For the `<document type>` key, you will have to use one of those listed in `/src/documents_types.js` (or create a new one there if it is not already referenced).
```

> You can find examples in `providers` folder.

Reviewer: ```suggestion
You can find examples in the `providers` folder.
```

> ## Deployment

Reviewer: ```suggestion
## Deploying
```
